modules/installation-and-upgrade/pages/container-deployment/mlm/server-air-gapped-deployment-mlm.adoc

```diff
@@ -51,3 +51,56 @@ transactional-update pkg install mgradm* mgrctl* suse-manager-5.0-$ARCH$-server-
 For more detailed information about installing {productname} Server on {sl-micro}, see xref:container-deployment/mlm/server-deployment-mlm.adoc[Deploy Server as a Virtual Machine].
 
 To upgrade {productname} Server, users should upgrade all packages in the system and follow the procedures defined in xref:container-management/updating-server-containers.adoc[Server Upgrade].
+
+== PTFs
+
+The PTF images are not available as packages.
+This means that they should be pulled using [command]``podman`` on a machine with internet access, then saved in an archive, transferred to the air-gapped machine and loaded there.
+
+.Procedure: Pulling the image on a machine with internet access
+. Install [command]``podman``.
+. Authenticate against the {suse} Registry using the SCC credentials:
+
++
+
+[source,shell]
+----
+set +o history
+echo SCC_MIRRORING_PASSWORD | podman login -u "SCC_MIRRORING_USER" --password-stdin registry.suse.com
+set -o history
+----
+. Create a [path]``/tmp/ptf-images`` temporary file with the URL of the PTF images, one per line.
+In most of the cases only the server image is needed and it can be created with a command like the following, after replacing the [command]``SCC_USERID`` and [command]``PTFID`` values.
```

Comment:
```suggestion
  In most of the cases only the server image is needed and it can be created with a command like the following, after replacing the [command]``SCC_USERID`` and [command]``PTFID`` values.
```